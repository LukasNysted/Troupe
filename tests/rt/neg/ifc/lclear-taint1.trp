let val main = self () 
    fun f1 () = send (main, 1); send (main, 2)
    val _ = spawn (f1)
    val c = raisembox `{alice}`
    val _ = debugpc()
    val u = rcv (`{}`, `{alice}`, [hn x => x]) 
    val _ = debugpc ()
    val v = rcv (`{}`, `{}`, [hn x => x ])
in adv v
end