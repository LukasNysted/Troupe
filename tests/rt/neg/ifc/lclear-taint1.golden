2020-03-16T16:32:07.462Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
[31mPID:7805410f-4970-4777-bd1e-49aa041c54b2@{}%{}    [39m [31mPC:{}               [39m [31mBL:{}               [39m [31mHNNORMAL            [39m 
[31mPID:7805410f-4970-4777-bd1e-49aa041c54b2@{}%{}    [39m [31mPC:{}               [39m [31mBL:{alice}          [39m [31mHNNORMAL            [39m 
[31mRuntime error in thread 7805410f-4970-4777-bd1e-49aa041c54b2@{}%{}[39m
[31m>> Illegal flow in adv function:[39m
[31m |    pc: {}[39m
[31m | block: {alice}[39m
[31m | value: 2@{alice}%{alice}[39m
